### PR TITLE
Quiz display template changes 

### PR DESCRIPTION
**Pull request by adityacp**
Along with template changes also added test cases for python traceback errors

### CODE REVIEW

**Comment by ankitjavalkar** [comment]
Looks good!

File: yaksh/evaluator_tests/test_python_evaluation.py [hunk 1]
```diff
@@ -2,52 +2,115 @@
...
     def test_correct_answer(self):
-        user_answer = "def add(a, b):\n\treturn a + b"""
-        get_class = PythonCodeEvaluator(self.test_case_data, self.test, self.language, user_answer, ref_code_path=None, in_dir=None)
-        result = get_class.evaluate()
+        user_answer = dedent("""
+        def add(a, b):
+            return a + b
+        """)
+        get_evaluator = PythonCodeEvaluator(self.test_case_data, self.test, self.language, user_answer, ref_code_path=None, in_dir=None)
+        result = get_evaluator.evaluate()
```
Comment (prabhuramachandran) [inline yaksh/evaluator_tests/test_python_evaluation.py]: Wrap lines to 80 cols.  Why are you passing the ref_code_path etc. when they are the default values!  Remove them.

File: yaksh/evaluator_tests/test_python_evaluation.py [hunk 2]
```diff
@@ -2,52 +2,115 @@
...
+        """)
+        indent_error_msg = "IndentationError"
+        get_evaluator = PythonCodeEvaluator(self.test_case_data, self.test, self.language, user_answer, ref_code_path=None, in_dir=None)
+        result = get_evaluator.evaluate()
+        self.assertFalse(result.get("success"))
+        self.assertIn(indent_error_msg, result.get("error"))
+
+    def test_name_error(self):
+        user_answer = ""
+        name_error_msg = "NameError"
```
Comment (prabhuramachandran) [inline yaksh/evaluator_tests/test_python_evaluation.py]: So this test will pass even if the error was simply NameError with no backtrace information!  Similarly for the others.

Comment (adityacp) [inline yaksh/evaluator_tests/test_python_evaluation.py]: Yes sir

File: yaksh/evaluator_tests/test_python_evaluation.py (hunk 1 above)
Comment (adityacp) [inline yaksh/evaluator_tests/test_python_evaluation.py]: This class call was written earlier. I will change it with next commit

File: yaksh/evaluator_tests/test_python_evaluation.py (hunk 2 above)
Comment (prabhuramachandran) [inline yaksh/evaluator_tests/test_python_evaluation.py]: Yes, I do know that, I was pointing out that your test should probably do more than just this!

**Comment by prabhuramachandran** [comment]
OK LGTM.

**Pull request merged**